assets/tablet.css

```diff
@@ -0,0 +1,286 @@
+@media screen and (min-width: 768px) and (max-width: 1023px) {
+  .hide {
+    display: none;
+  }
+
+  main {
+    height: 590.2vh;
+  }
+
+  /* nav-bar */
+
+  .non-fixed2 {
+    display: none;
+  }
+
+  .fixed1 ul {
+    display: flex;
+    position: relative;
+    left: -4.08vw;
+    top: -20px;
+  }
+
+  .non-fixed {
+    width: 85%;
+    float: right;
+    height: 3.8vh;
+    align-items: center;
+    background-color: #000;
+  }
+
+  .fixed1 {
+    display: flex;
+    width: 100%;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] You already have `display:flex` pro[erty, please use it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags


**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_